projects/js-packages/components/components/status/style.module.scss

```diff
@@ -0,0 +1,63 @@
+.status {
+    --font-size: var( --font-body-extra-small ); // 12px
+    --indicator-size: calc( var( --font-size ) * 0.666 ); // 8px
+
+    align-items: center;
+    display: flex;
+    font-size: var( --font-size );
+    font-weight: 600;
+    line-height: 1.666;
+    white-space: nowrap;
```

Comment:
Any reason not to use `Text` component? We should avoid set directly the `font-size` and use the `Text` components, since it contains the rules for our fonts.

Reply:
Thanks for catching this, no reason to not use `Text`: https://github.com/Automattic/jetpack/pull/30209/commits/0b7c845d0def850fe5099d654fe2371ad25a17b2 :+1: